File: 1678_Goal_Parser_Interpretation.java
class Solution {
    public String interpret(String command) {
        String str = "";
        for (int i = 0; i < command.length(); i++) {
            if (command.charAt(i) == 'G') {
                str += "G";
            } else {
                if (command.charAt(i) == '(') {
                    if (command.charAt(i + 1) == 'a') {
                        str += "al";
                    } else {
                        str += "o";
                    }
                }
            }
        }
        return str;
    }
}

File: 1678_Goal_Parser_Interpretation.cs
public class Solution {
    public string Interpret(string command) {
        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < command.Length; i++) {
            if (command[i] == 'G') sb.Append("G");
            else if (command[i] == '(' && command[i + 1] == 'a') sb.Append("al");
            else if (command[i] == '(') sb.Append("o");
        }
        return sb.ToString();
    }
}